### ISSUE TITLE
Show Satismeter survey after app launch

### ISSUE THREAD

**Issue by Neil3079** [satismeter/satismeter-ios#16]
My application shows an initial loading screen whilst data is downloaded before transitioning to the first screen of my app. When I don't call identify in didFinishLaunching Satismeter never shows and I can't clearly see a way to launch it at another point on the customer journey

**Comment by Neil3079** [comment]
Further investigation has revealed that the survey is being presenter but only on the root view controller of the application, is there anyway to specify which view the survey should be added to?

**Comment by jsedlacek** [comment]
Thanks @Neil3079 for the report. Let me check if there is a way to specify other view controller.

**Comment by jsedlacek** [comment]
At the moment there is no way to change to other than root view controller of the application.
We will add an option to change this in future releases. I am keeping this issue open until it's implemented.

**Comment by jamesjara** [comment]
@jsedlacek do you have ETA for this fix?

**Comment by jsedlacek** [comment]
@jamesjara sorry, we don't have ETA at the moment.

We will start working on a new release once we collect enough improvements. I will update this ticket as we have more details.

**Comment by jamesjara** [comment]
I'm working in another client, and two years latter asking the same question, any update ?  :) thanks guys

**Comment by jsedlacek** [comment]
@jamesjara I am afraid we don't have any update on this. Is this blocking you from implementing SatisMeter? Could you please describe your use-case so I can check with the dev team?

**Comment by jamesjara** [comment]
not really, I did a workaround, but it would be nice to have a method to set up and another method to force the visibility, like showing intentionally the widget
